cmds/strings/strings.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Strings finds printable strings.
+//
+// Synopsis:
+//     strings OPTIONS [FILES]...
+//
+// Description:
+//     Prints all sequences of `n` or more printable characters terminated by a
+//     non-printable character (or EOF).
+//
+//     If no files are specified, read from stdin.
+//
+// Options:
+//     -n number: the minimum string length (default is 4)
+package main
+
+import (
+	"bufio"
+	"io"
+	"log"
+	"os"
+
+	flag "github.com/spf13/pflag"
+)
+
+var (
+	n = flag.Int("n", 4, "the minimum string length")
+)
+
+func asciiIsPrint(char byte) bool {
```

Comment:
I assume strconv.IsPrint was not really the right thing ...

Reply:
`strconv.IsPrint` has an algorithm for Unicode. We just need something simple (and fast) for ASCII.